.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
We don't need to commit the ./idea folder

Reply:
deleted

Reply:
its still here

Reply:
This is auto generated by Jetbrains IDEs (IntellJ, Android Studio etc). You will need to add `.idea` to your git ignore